docs/storage-provider/how-providing-works.md

```diff
@@ -0,0 +1,199 @@
+---
+title: 'How providing storage works'
+description: 'Providing storage to the Filecoin network involves accepting deals and honoring them by inserting proofs in the chain.'
+---
+
+# {{ $frontmatter.title }}
```

Comment:
```suggestion
```

We don't use `$frontmatter.title` anymore, since we moved over to the Hugo platform.